src/db/queries/postQueries.js

```diff
@@ -0,0 +1,17 @@
+const connect = require('../db_connections.js');
+
+const getHomePagePosts = (cb) => {
+  connect.query(
+    `(SELECT * FROM posts where type = 'story' order by time_stamp desc LIMIT 2)
+      UNION ALL
+      (SELECT * FROM posts where type = 'advise' order by time_stamp desc LIMIT 2)
```

Comment:
can you explain whats going on in this query?

Reply:
creating three selects at the same time, everyone is different.

Reply:
as in im guessing you are getting three types of posts from the database